File: Contest-1/Python/Que2.py
# Approach -> Given, the positions of two hunters and a rabbit. We need to find absolute value of distance b/w hunters and rabbit, and compare them.
# Read input integers
x, y, z = input().split()
x = int(x)
y = int(y)
z = int(z)

# Calculate distances
a = abs(x - z)
b = abs(y - z)

# Compare distances and print the result
if a == b:
    print('C')
elif a > b:
    print('B')
else:
    print('A')

File: Contest-1/C++/Que2.cpp
// Approach -> Given, the positions of two hunters and a rabbit. We need to find absolute value of distance b/w hunters and rabbit, and compare them.

#include <bits/stdc++.h>
using namespace std;

int main(){
    int x, y, z;
    cin >> x >> y >> z;
    int a = abs(x-z);
    int b = abs(y-z);
    // abs gives the modulus or absolute value.
    if (a == b) cout << 'C';
    else if (a > b) cout << 'B';
    else    cout << 'A';
}
